setup.py

```diff
@@ -42,7 +42,7 @@ def find_version(*file_paths):
 long_description = read('README.rst')
 
 tests_require = ['pytest', 'virtualenv>=1.10', 'scripttest>=1.3', 'mock',
-                 'pretend']
+                 'pretend', 'pytest-catchlog', 'freezegun']
```

Comment:
I did for some reason I don't recall (possibly because I had to install something by hand to make pytest work). I can revert it. Do you want a new PR for this?